django/contrib/admindocs/views.py

```diff
@@ -60,11 +55,11 @@ class TemplateTagIndexView(BaseAdminDocsView):
     template_name = 'admin_doc/template_tag_index.html'
 
     def get_context_data(self, **kwargs):
-        load_all_installed_template_libraries()
-
+        # Non-trivial TEMPLATES settings aren't supported (#24125).
+        engine = Engine.get_default()
```

Comment:
If I understand correctly, there's no way to reach this view through the admindocs UI when `Engine.get_default()` raises `ImproperlyConfigured`, so you don't need to catch the exception. Is that correct?